libraries/botbuilder-dialogs-adaptive/src/conditions/onCommandActivity.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ActivityTypes } from 'botbuilder';
+import { Dialog } from 'botbuilder-dialogs';
+import { OnActivity } from './onActivity';
+
+/**
+ * Actions triggered when a Command activity is received.
+ */
+export class OnCommandActivity extends OnActivity {
+    public static $kind = 'Microsoft.OnCommandActivity';
+
+    /**
+     * Intiializes a new instance of the [OnCommandActivity](xref:botbuilder-dialogs-adaptive.OnCommandActivity) class.
+     *
+     * @param actions Optional, list of actions.
+     * @param condition Optional, condition which needs to be met for the actions to be executed.
+     */
+    public constructor(actions?: Dialog[], condition?: string) {
```

Comment:
I don't think we use `public` on the constructors, at the very least we're not consistent... @joshgummersall since the default is `public`, what do you want to follow as a policy going forwards?

Reply:
Sorry - merged this before you had a chance to comment. I'm typically in favor of being terse whenever possible, so I tend to omit `public` since it is the default. I'm open to either, though, and I agree we should use a consistent style.

Reply:
Filed https://github.com/microsoft/botbuilder-js/issues/3518 to track removing `public` where unnecessary and adding a corresponding eslint rule.